.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -1,24 +1,43 @@
 ---
 name: Bug report
-about: File a bug report in OJS, OMP or OPS. Try our support forum linked below if you can't provide reproduction steps and technical specifications.
-title: ''
+about: File a bug report in OJS, OMP or OPS. Try our Software Support forum, linked below, if you cannot provide reproducible steps and technical specifications.
+title: '[Main Feature] | Workflow Section - Page, Description of bug'
 labels: ''
 assignees: ''
 
 ---
 
-**Describe the bug**
-Please tell us what happens, what you expected to happen, and why you think it is a bug in the software.
+**Description:**
+A detailed explanation of the bug, include any unusual behavior or unique details.  This is the chance to provide some context surrounding the bug.
 
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+**Preconditions or Prerequisites:**
+These are requirements that need to be in place before trying to follow the steps to reproduce bug behavior.  If necessary, it is helpful to include user details, related plugins, submission workflow stage etc... Sometimes the best precondition is that it is seen using "basic install with default dataset"
 
+**Steps to Reproduce:**
+Clear, unambiguous, step-by-step instructions on how to trigger the bug.
+1.  Login with User <role>
+2.  Select Active Submissions using side navigation views
+3.  Select View button on a submission in the Review Stage (example id 14)
+4.  Within the submission view select Publication -> Galleys from submission navigation
+5.  Select Add Galley Button
+6.  Fill out the add Galley Form
+a.  Sub-step 1 to complete main step
+b.  Fill in Galley Label field
+c.  Fill in the Language (ex English) from drop down
+d.  Leave Galley will be available Checkbox unchecked.
+e.  Leave URL link field empty
+6.  Select Save button
+
+**Expected Result:** Describe what the system should do under normal conditions.
+**Actual Result:** Describe what the system actually does (the problematic behavior).
+
+**Repro Rate:** Usually given as a rough percentage of how many times going through the steps this bug is seen.  (ex. less than 50%, greater than 50%, 100%, and intermittent) 
```

Comment:
Should we expect external users to estimate this, or is this more for internal use? Perhaps it could be 3 options instead of a percentage, or just a question like "Does this issue occur consistently?".